Firestore/Source/API/FIRQuery.mm

```diff
@@ -538,38 +553,42 @@ - (Bound)boundFromSnapshot:(FIRDocumentSnapshot *)snapshot isBefore:(BOOL)isBefo
   }
   const Document &document = *snapshot.internalDocument;
   const DatabaseId &databaseID = self.firestore.databaseID;
-  std::vector<FieldValue> components;
+  const OrderByList &order_bys = self.query.order_bys();
+
+  google_firestore_v1_ArrayValue components;
+  components.values_count = CheckedSize(order_bys.size());
+  components.values = MakeArray<google_firestore_v1_Value>(components.values_count);
```

Comment:
I will consider this if there are more callsites where I can't use SetRepeatedField. I think this should be limited to the iOS Objective-C code, since it uses datastructures that are not compatible with the current method.